src/wscearth/views.py

```diff
@@ -75,12 +98,22 @@ def positions():
     #    print(row)
 
     if len(df) == 0:
-        return flask.render_template("positions_map.html",
-                                    centre_lat=-25.0,
-                                    centre_long=130.0,
-                                    df=[])
-
-    return flask.render_template("positions_map.html",
-                                    centre_lat=df["latitude"].mean(),
-                                    centre_long=df["longitude"].mean(),
-                                    df=df)
+        df = []
+        center = {
+            "lat": -25.0,
+            "lng": 130.0,
+        }
+    else:
+        center = {
+            "lat": df["latitude"].mean(),
+            "lng": df["longitude"].mean(),
+        }
+
+    # We're in 2023 and this the best Python can do?
+    items = json.loads(df.to_json(orient="records"))
+
+    return json.dumps({
+        "center": center,
```

Comment:
nit: We be aussie. :-)

Reply:
Yes, it pains me to type it. But America conquered software long ago. The Queen's English has been relegated to the i18n config for now.